core/modules/checkout/store/checkout/actions.ts

```diff
@@ -6,65 +6,74 @@ import { Logger } from '@vue-storefront/core/lib/logger'
 import { StorageManager } from '@vue-storefront/core/lib/storage-manager'
 
 const actions: ActionTree<CheckoutState, RootState> = {
-  /**
-   * Place order - send it to service worker queue
-   * @param {Object} commit method
-   * @param {Object} order order data to be send
-   */
-  async placeOrder ({ state, commit, dispatch }, { order }) {
+  async placeOrder ({ dispatch }, { order }) {
     try {
-      const result = await dispatch('order/placeOrder', order, {root: true})
+      const result = await dispatch('order/placeOrder', order, { root: true })
       if (!result.resultCode || result.resultCode === 200) {
-        StorageManager.get('user').setItem('last-cart-bypass-ts', new Date().getTime())
-        await dispatch('cart/clear', { recreateAndSyncCart: true }, {root: true})
-        if (state.personalDetails.createAccount) {
-          commit(types.CHECKOUT_DROP_PASSWORD)
-        }
+        await dispatch('saveBypassTs')
+        await dispatch('cart/clear', { recreateAndSyncCart: true }, { root: true })
+        dispatch('dropPassword')
       }
     } catch (e) {
       Logger.error(e, 'checkout')()
     }
   },
+  async saveBypassTs () {
+    const userStorage = StorageManager.get('user')
+    await userStorage.setItem('last-cart-bypass-ts', new Date().getTime())
+  },
+  dropPassword ({ commit, state }) {
```

Comment:
just in case we'd like to add something here in future
```suggestion
  async dropPassword ({ commit, state }) {
```